src/main/webapp/js/community-page-loader.js

```diff
@@ -1,40 +1,52 @@
       /** Fetches users and adds them to the page. */
-      function fetchUserList() {
+      function fetchUserList() 
+      {
         const url = '/user-list';
         fetch(url).then((response) => {
           return response.json();
         }).then((users) => {
           const list = document.getElementById('list');
           list.innerHTML = '';
-
+		  
+          /*separating user link and about me section by " " */
           users.forEach((user) => {
-            let EndOfUsernameindex = user.indexOf(".com")+4;
-            const userListItem = buildUserListItem(user.substring(0, EndOfUsernameindex), user.substring(EndOfUsernameindex));
+            let endOfUsernameindex = user.indexOf(" ");
+            const userListItem = buildUserListItem(user.substring(0, endOfUsernameindex), user.substring(endOfUsernameindex+1));
             list.appendChild(userListItem);
           });
         });
       }
 
       /**
-       * Builds a list element that contains a link to a user page, e.g.
-       * <li><a href="/user-page.html?user=test@example.com">test@example.com</a></li>
+       * Builds a list element that contains:
+       * a link to a user page, e.g. and 
+       * displays user "about me" text if posted by user
+       * example:
+       * <li><a href="/user-page.html?user=test@example.com">test@example.com</a>
+       * <textarea readonly = 'readonly'> about me <textarea></li>
        */
-      function buildUserListItem(user, aboutMe) {
+      function buildUserListItem(user, aboutMe) 
+      {
+        const userListItem = document.createElement('li');
+		
         const userLink = document.createElement('a');
         userLink.setAttribute('href', '/user-page.html?user=' + user);
         userLink.appendChild(document.createTextNode(user));
-	const aboutMeParagraph = document.createElement('textarea');
-	aboutMeParagraph.setAttribute('placeholder', 'About me ...');
-	aboutMeParagraph.setAttribute('readonly', 'readonly');
-	aboutMeParagraph.innerHTML = aboutMe;
-        const userListItem = document.createElement('li');
         userListItem.appendChild(userLink);
-	userListItem.appendChild(document.createElement('br'));
-	userListItem.appendChild(aboutMeParagraph);
+        userListItem.appendChild(document.createElement('br'));
+		
+        if(aboutMe!==null && aboutMe!=="")
```

Comment:
You can also use replace with if(aboutMe)
https://stackoverflow.com/questions/154059/how-to-check-empty-undefined-null-string-in-javascript

Reply:
That's nice and clean!